autoload/gh/issues.vim

```diff
@@ -194,17 +194,31 @@ function! gh#issues#new() abort
   call gh#gh#init_buffer()
   call gh#gh#set_message_buf('loading')
 
-  let m = matchlist(bufname(), 'gh://\(.\{-}\)/\(.\{-}\)/\(.*\)/issues/new$')
+  let m = matchlist(bufname(), 'gh://\(.\{-}\)/\(.\{-}\)/issues/new$')
+
+  let l:owner = m[1]
+  let l:repo = m[2]
   let s:gh_issue_new = {
-        \ 'owner': m[1],
-        \ 'name': m[2],
-        \ 'branch': m[3],
+        \ 'owner': l:owner,
+        \ 'name': l:repo,
+        \ 'branch': '',
```

Comment:
I think it's better to define default value `main`.

Reply:
thank, i didn't know about that ^^
fixed by [this](https://github.com/skanehira/gh.vim/pull/87/commits/2e455667b44c10fa3871b4b4453ed4b4aa9cf658).

Reply:
fixed by [this](https://github.com/skanehira/gh.vim/pull/87/commits/e733b04dc85272156a6092880693747233679a7c) commit.